<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow3D</class>
 <widget class="QMainWindow" name="MainWindow3D">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>998</width>
    <height>757</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow3D</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="Viewer" name="viewerFuncO"/>
      </widget>
      <widget class="QSplitter" name="splitter_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="Viewer" name="viewerFunc"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>998</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <widget class="QMenu" name="menu_Connection">
    <property name="title">
     <string>&amp;Connection</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
   </widget>
   <widget class="QMenu" name="menu_Settings">
    <property name="title">
     <string>&amp;Settings</string>
    </property>
    <addaction name="actionNetwork"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Connection"/>
   <addaction name="menu_Settings"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionConnect"/>
   <addaction name="actionNetwork"/>
   <addaction name="separator"/>
   <addaction name="actionHR"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="3dhomeo.qrc">
     <normaloff>:/icons/resource/fileopen.png</normaloff>:/icons/resource/fileopen.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="3dhomeo.qrc">
     <normaloff>:/icons/resource/Network_Globe_Connected_Icon_256.png</normaloff>:/icons/resource/Network_Globe_Connected_Icon_256.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Connect</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="icon">
    <iconset resource="3dhomeo.qrc">
     <normaloff>:/icons/resource/Network_Globe_Disconnected_Icon_256.png</normaloff>:/icons/resource/Network_Globe_Disconnected_Icon_256.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Disconnect</string>
   </property>
  </action>
  <action name="actionNetwork">
   <property name="icon">
    <iconset resource="3dhomeo.qrc">
     <normaloff>:/icons/resource/Network Connection Internet.png</normaloff>:/icons/resource/Network Connection Internet.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Network</string>
   </property>
  </action>
  <action name="actionHR">
   <property name="text">
    <string>High Resolution</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Generate the high resolution volume&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QGLViewer</class>
   <extends>QWidget</extends>
   <header>QGLViewer/qglviewer.h</header>
  </customwidget>
  <customwidget>
   <class>Viewer</class>
   <extends>QGLViewer</extends>
   <header>viewer.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="3dhomeo.qrc"/>
 </resources>
 <connections/>
</ui>
